Split version increments into related and unrelated to the API (#83)
AMWA-TV/nmos-sink-metadata-processing: docs/Interoperability.md

## Version Increments

> In order to prevent unnecessary polling of the Flow Compatibility Management API, changes to it are signalled via the IS-04 versioning mechanism. E.g. when the Input/Output parameters of a Sender/Receiver are modified, when set of Inputs/Outputs associated with the Sender/Receiver is changed, or when Status of Sender/Receiver is updated, then the `version` attribute of the relevant IS-04 Sender/Receiver MUST be incremented.

Reviewer: I would not  use "changes to it" ... but simply "changes"

I would add "when state of Inputs/Outputs associated with the Sender/Receiver changes"

I would rephrase "In order to prevent unnecessary polling of the Flow Compatibility Management API, changes are signaled via the IS-04 versioning mechanism of the associated Sender and Receiver objects"

Author: Thanks, it sounds better.

```suggestion
In order to prevent unnecessary polling of the Flow Compatibility Management API, changes are signaled via the IS-04 versioning mechanism of the associated Sender and Receiver resources. E.g. when the Input/Output parameters of a Sender/Receiver are modified, when set of Inputs/Outputs associated with the Sender/Receiver is changed, or when Status of Sender/Receiver is updated, then the `version` attribute of the relevant IS-04 Sender/Receiver MUST be incremented.
```

Reviewer: Maybe "properties" instead of "parameters" ... ?

Author: ```suggestion
In order to prevent unnecessary polling of the Flow Compatibility Management API, changes are signaled via the IS-04 versioning mechanism of the associated Sender and Receiver resources. E.g. when properties of any Input/Output associated with the Sender/Receiver are modified, when set of Inputs/Outputs associated with the Sender/Receiver is changed, or when Status of Sender/Receiver is updated, then the `version` attribute of the relevant IS-04 Sender/Receiver MUST be incremented.
```

> If any resource such as Source, Flow, Sender etc. changes over its lifetime, then its `version` attribute MUST be incremented as well.

Reviewer: I would rephrase "If the state of any resource such as Source, Flow, Sender, Receiver, etc. changes over its lifetime, then its `version` attribute MUST be incremented as well."

Author: It seems more confusing to me because we use term "state" for Sender/Receiver. What about "properties"?

```suggestion
If properties of any resource such as Source, Flow, Sender, Receiver, etc. change over its lifetime, then its `version` attribute MUST be incremented as well.
```

Reviewer: ok
